.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "Nop/nopCommerce"]
+	path = Nop/nopCommerce
```

Please create a Jira ticket to change this to just `nopCommerce`.

```suggestion
	path = nopCommerce
```